definitions/features/candlepin_database.rb

```diff
@@ -48,7 +49,7 @@ def env_content_ids_with_null_content
   def load_configuration
     raw_config = File.read(CANDLEPIN_DB_CONFIG)
     full_config = Hash[raw_config.scan(/(^[^#\n][^=]*)=(.*)/)]
-    uri_regexp = %r{://(([^/:]*):?([^/]*))/([^?]*)\??(ssl=([^&]*))?}
+    uri_regexp = %r{://(([^/:]*):?([^/]*))/([^?]*)\??(ssl=([^&]*))?\&?(sslfactory=([^&]*))?}
```

Comment:
This depens on the ordering of the params (the `ssl` and  `sslfactory`). It would be probably better not trying to extract them both in the same regexp, but having separate regexps for those two.

Reply:
Done!